terraform/vpc-app.tf

```diff
@@ -0,0 +1,21 @@
+module "app-vpc" {
```

Comment:
Should this be in its own module, separate from the management VPC? Thinking that will be the easiest way to set up multiple of them, but I'm not entirely sure on best practices here.

Also, mind if we use underscores instead of hyphens for all references like this? Ditto for the other resources etc. in this pull request.

Reply:
Yep, probably should be it's own module. Also, should setup the peering between the mgmt VPC and itself.  Example in cloud.gov: https://github.com/18F/cg-provision/blob/master/terraform/modules/stack/spoke/spoke.tf

Reply:
Yeah, forgot the peering... will get this in over the weekend or Monday.

Reply:
> mind if we use underscores instead of hyphens for all references like this?
> ...
> probably should be it's own module

Bump on these 😸